packages/webview_flutter/example/pubspec.yaml

```diff
@@ -17,6 +17,7 @@ dependencies:
     path: ../
 
 dev_dependencies:
+  espresso: ^0.1.0+2
```

Comment:
I don't think this is used. is it?

Reply:
It's not in this PR, but I added it as part of "adding the test harness". I assumed that when another user wanted to create a PR and needed to add a test, that we didn't want to require them to add dependencies.

Reply:
Sounds good